packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -312,9 +318,13 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
       const currentVotes = await Promise.all(
         voteStatuses.map(async (voteStatus, index) => {
           if (voteStatus.committedValue) {
-            return JSON.parse(
-              await decryptMessage(decryptionKeys[account][currentRoundId].privateKey, voteStatus.committedValue)
-            );
+            try {
+              return JSON.parse(
+                await decryptMessage(decryptionKeys[account][currentRoundId].privateKey, voteStatus.committedValue)
```

Comment:
Does this still fail since we last talked? Catching this error might confuse users in case the messages cannot be decrypted, but I suppose its better than crashing the App.

Reply:
This was failing for me until I bumped the eth-crypto version from 1.6.0 to 1.7.0 in the common package. I dont know if this will break other things, but doesnt seem like it so far.  I left the try catch in since this wasnt really being handled.

Reply:
Got it. I'm OK with leaving that here so the App doesn't crash, but its still going to be jarring for the user if they can't see their committed value (they will wonder, "did my commit go through correctly"?)

Reply:
yea thats a great point but im not aware of any infrastructure within the voter dapp to alert the user to errors. if you have a suggestion on how to do this LMK